website/layouts/index.html

```diff
@@ -0,0 +1,78 @@
+            <img class="img-fluid" style="max-width: 60%" src="Thanos-logo_full.svg" alt="Thanos Logo"/>
+        </div>
+    </div>
+    <div class="row py-3 pb-lg-5">
+        <div class="col text-center">
+            <h3 class="mb-3">Highly available Prometheus setup with long term storage capabilities.</h3>
+            <ul class="list-inline mb-0">
+                <li class="list-inline-item my-3">
+                    <a href="{{ "getting-started.md" | relURL }}" class="btn btn-outline-secondary">
+                        <i class="fas fa-fw fa-book"></i> Getting Started
+                    </a>
+                </li>
+                <li class="list-inline-item my-3">
+                    <a href="https://github.com/{{ .Site.Params.GithubUser }}/{{ .Site.Params.GithubProject }}/releases" class="btn btn-outline-secondary">
+                        <i class="fas fa-fw fa-download"></i> Download
+                    </a>
+                </li>
+                <li class="list-inline-item my-3">
+                    <a href="https://github.com/{{ .Site.Params.GithubUser }}/{{ .Site.Params.GithubProject }}" class="btn btn-outline-secondary">
+                        <i class="fab fa-fw fa-github"></i> Github
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</div>
+<div class="container-fluid bg-light">
+    <div class="row">
+        <div class="col-12 col-md-6 col-lg text-center py-5">
+            <i class="fas fa-search fa-5x db mb-3"></i>
+            <h4 class="font-weight-bold">Global Query View</h4>
+            <p>Scale your Prometheus setup by enabling querying your Prometheus metrics across multiple Prometheus server and clusters.</p>
+        </div>
+        <div class="col-12 col-md-6 col-lg text-center py-5">
+            <i class="fas fa-archive fa-5x db mb-3"></i>
+            <h4 class="font-weight-bold">Unlimited Retention</h4>
+            <p>Extend the system with the object storage of your choice to store your metrics for unlimited time. Supports GCP, S3, Azure, Swift and Tencent COS.</p>
+        </div>
+        <div class="col-12 col-md-6 col-lg text-center py-5">
+            <i class="fas fa-plug fa-5x db mb-3"></i>
+            <h4 class="font-weight-bold">Prometheus Compatible</h4>
+            <p>Use the same tools you love such as Grafana or others that supports Prometheus Query API.</p>
+        </div>
+        <div class="col-12 col-md-6 col-lg text-center py-5">
+            <i class="fas fa-file-download fa-5x db mb-3"></i>
+            <h4 class="font-weight-bold">Downsampling & Compaction</h4>
+            <p>Downsample historical data for massive query speedup when querying large time ranges or configure complex retention policies.</p>
+        </div>
+    </div>
+</div>
+<div class="container-fluid">
+    <div class="row py-5">
+        <div class="col text-center">
+            <h3 class="color-purple">Join the community !</h3>
```

Comment:
Small nit but there probably shouldn't be a space before `!`

Reply:
damn, forget this, we will fix in later Pr :+1:

Reply:
plus markdown bit ):